src/Caching/Hybrid/src/HybridCacheBuilderExtensions.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Caching.Hybrid;
+
+/// <summary>
+/// Configuration extension methods for <see cref="IHybridCacheBuilder"/> / <see cref="HybridCache"/>
```

Comment:
Nit: Full stops at the end of all your summary sentences.

Reply:
done.